spec/models/govbox_set_spec.rb

```diff
@@ -167,71 +167,38 @@
           allow(affiliate).to receive(:agency).and_return(agency)
         end
 
-        it "should call Jobs.search with the query, org codes, size, hl, and lat_lon params" do
+        it "should call Jobs.search with the query, org codes, results per page, and location name params" do
           expect(Jobs).to receive(:search).
-            with(:query => 'foo', :hl => 1, :size => 10, :organization_ids => 'ABCD,BCDE', :lat_lon => '12.34,-34.56').
+            with(query: 'foo',
+                 ResultsPerPage: 10,
+                 Organization: 'ABCD,BCDE',
+                 LocationName: 'Washington, DC').
             and_return(job_openings)
-          govbox_set = GovboxSet.new('foo', affiliate, geoip_info)
-          expect(govbox_set.jobs.first.position_title).to eq('<strong>Nurse</strong>')
+          govbox_set = GovboxSet.new('foo', affiliate, location_name)
+          expect(govbox_set.jobs.first.position_title).to eq('<em>Nurse</em>')
```

Comment:
nope passes.  returns it with default <em>Nurse</em> they must have the default highlighting on.

Reply:
oh - it looks like it's only passing because we're stubbing the return value of the `Jobs.search`, and the stubbed values include highlighting:
https://github.com/GSA/search-gov/pull/159/files#diff-306a6bc4d0ae572bc309be3facc23fcdL150

We should make sure that the stubbed responses in this spec reflect the JSON format returned by USAJOBs.